src/sentry/api/endpoints/group_tags.py

```diff
@@ -24,7 +24,7 @@ def get(self, request, group):
         else:
             value_limit = 10
 
-        use_snuba = request.GET.get('enable_snuba') == '1'
+        use_snuba = use_snuba = options.get('snuba.events-queries.enabled')
```

Comment:
So one question about this part. Should this be under a different flag? the `snuba.events-queries.enabled` flag seems is still pretty much reversible as the "old" implementation of Events still works fine (as evidenced by the fact we've turned it off an on about 10 times in the last week 😄 ) but I'm not sure we want to put this tagstore related stuff on the same switch? What happens to this endpoint when it's switched off?

Reply:
Oh I guess I see, it would just use the first environment id `environment_ids[:1]` and the switch for which tagstore implementation to use is already on a different switch, namely the `SENTRY_TAGSTORE` config option.